internal/definitions/definition_handler_identity_update.go

```diff
@@ -48,8 +48,17 @@ func (dh *definitionHandlers) handleIdentityUpdateBroadcast(ctx context.Context,
 		return HandlerResult{Action: ActionReject}, nil
 	}
 
+	parent, retryable, err := dh.identity.VerifyIdentityChain(ctx, identity)
+	if err != nil && retryable {
+		return HandlerResult{Action: ActionRetry}, err
+	} else if err != nil {
+		log.L(ctx).Infof("Unable to process identity update (parked) %s: %s", msg.Header.ID, err)
```

Comment:
Why is this one parked? This would be for things like an unknown or unconfirmed parent identity. All other failures here seem to result in outright rejection of the update.

Reply:
In this case, we couldn't resolve something up the parent stack. The code in creation (which this was copied from) did a wait at this point, on the basis there could be something in the parent stack still being resolved.

I believe it's technical also possible when replaying a chain for that to be true here. Or at least I've not done enough analysis of the topics used on the messages in that scenario to rule it out. So I left the logic equivalent in create and update paths.